net/coeurl/Portfile

```
@@ -24,6 +24,14 @@ meson.wrap_mode     default
 # Fix subproject wrap file for curl: [provides] should be [provide]
 patchfiles-append   patch-subproj-curl.diff
 
+# Backport of:
+# https://github.com/gabime/spdlog/commit/c65aa4e4889939c1afa82001db349cac237a13f8
+pre-build {
+    set spdlog_v    1.8.5
+    system -W ${worksrcpath}/subprojects/spdlog-${spdlog_v} \
+        "/usr/bin/patch -p0 < ${filespath}/patch-pthread.diff"
```

Variables passed to the shell should be escaped.
```suggestion
        "/usr/bin/patch -p0 < [shellescape ${filespath}]/patch-pthread.diff"
```